src/Exception/FilesystemFailureException.php

```diff
@@ -59,6 +66,11 @@ public static function forFailedWriteOperation(string $path): self
         return new self(sprintf('An error occurred when writing file contents to "%s".', $path), 1639415423);
     }
 
+    public static function forFailedCommandExecution(string $command): self
+    {
+        return new self(sprintf('An error occurred while executing "%s".', $command), 1670866484);
```

Comment:
Since `zip` doesn't seem to be supported at all, why not indicate that somewhere. I'm torn here. Restricting to `tarball` and/or gzipped `tarballs` seems easy but might leave people hanging.